python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -827,6 +833,10 @@ def deblend(self, mExposure, catalog):
                     catalog=catalog,
                     scarletSource=scarletSource,
                 )
+            # Log a message if it has been a while since the last log.
+            if (currentTime := time.time()) > logTime + self.config.loggingInterval:
```

Comment:
Rather than always comparing against `logTime + loggingInterval` every time, instead use `nextLogTime` instead of `lastLoggedTime` and calculate the next time at which you want to log and then simplify the if statement and only calculate the time with offset once you do log.

Reply:
That's an excellent suggestion! Adopted this now.